### PR TITLE
Fix in "UI" and "Machine learning"

### PR DESCRIPTION
**Pull request by virichino**
- Fix in the UI
- Delete commented code
- Update README
- Fix code in machine learning

### CODE REVIEW

File: MLendPoint/.babelrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "env": {
+        "test": {
+           "plugins": ["@babel/plugin-transform-modules-commonjs"]
+        }
+    }
+}
```
Comment (serqiiiio) [inline MLendPoint/.babelrc:7]: add the last line

File: MLendPoint/.gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 node_modules
+coverage
```
Comment (serqiiiio) [inline MLendPoint/.gitignore:2]: add the last line required

File: MLendPoint/src/controllers/MachineLearing.js [hunk 3]
```diff
@@ -18,14 +14,18 @@ class MachineLearing {
 
     constructor(imagePath, searchWord, percentage) {
         if (new.target === MachineLearing)
-            throw Error('Abstract class cannot be Instantiated');
+            throw new Error('MachineLearing abstract class cannot be instantiated');
         this.image = imagePath;
         this.searchWord = searchWord;
         this.percentage = percentage;
         this.predictions = undefined;
     }
```
Comment (serqiiiio) [inline MLendPoint/src/controllers/MachineLearing.js:23]: quit this extra space

File: MLendPoint/src/controllers/MachineLearing.js [hunk 4]
```diff
@@ -18,14 +14,18 @@ class MachineLearing {
...
 
-}
 
+    async getJSON(parameter) {
+        throw new Error('getJSON() must be implementrd');
+    }
+
```
Comment (serqiiiio) [inline MLendPoint/src/controllers/MachineLearing.js:28]: quit the extra space

File: MLendPoint/src/middleware/imageFilter.js [hunk 5]
```diff
@@ -1,42 +1,20 @@
...
-   
-/*fs.mkdir(path.join(__dirname, 'test'), (err) => {
-    if (err) {
-        return console.error(err);
-    }
-    console.log('Directory created successfully!');
-});*/
 
 const storage = multer.diskStorage({
     destination: function (req, file, cb) {
```
Comment (jchoque87) [inline MLendPoint/src/middleware/imageFilter.js:6]: You should use arrow function

File: MLendPoint/src/middleware/imageFilter.js [hunk 6]
```diff
@@ -1,42 +1,20 @@
...
         cb(null, __dirname + "/../../public/images");
     },
     filename: function (req, file, cb) {
```
Comment (jchoque87) [inline MLendPoint/src/middleware/imageFilter.js:9]: You should use arrow function

**Review by serqiiiio — CHANGES_REQUESTED** [review]
Require few clean code changes

File: MLendPoint/src/middleware/jsonResponse.js [hunk 7]
```diff
@@ -1,15 +1,15 @@
 const path = require("path");
 
 class JsonResponse {
-    response(result, algorithm, searchWord, secondsToString,image,response,pathImage) {
+    response(result, algorithm, searchWord, secondsToString, image, response, pathImage) {
         if (result[0] !== undefined) {
             const objectResult = result[0];
```
Comment (jchoque87) [inline MLendPoint/src/middleware/jsonResponse.js:6]: You should use all array

File: MLendPoint/src/middleware/obtainDirectory.js [hunk 8]
```diff
@@ -1,12 +1,12 @@
 const fs = require("fs");
 const path = require("path");
 
-class ObtainDirectory{
-    constructor() {}
+class ObtainDirectory {
+    constructor() { }
```
Comment (jchoque87) [inline MLendPoint/src/middleware/obtainDirectory.js:5]: This constructor don't have any arguments. You should remove  this constructor if you don't use it

File: MLendPoint/src/middleware/jsonResponse.js [hunk 9]
```diff
@@ -1,15 +1,15 @@
 const path = require("path");
 
 class JsonResponse {
-    response(result, algorithm, searchWord, secondsToString,image,response,pathImage) {
+    response(result, algorithm, searchWord, secondsToString, image, response, pathImage) {
```
Comment (jchoque87) [inline MLendPoint/src/middleware/jsonResponse.js:4]: Parameter searchWord is not used in this function you should remove this parameter

File: ReactUI/src/ui/components/materialUI/machine-learning/TableML.jsx [hunk 10]
```diff
@@ -19,40 +19,40 @@ const TableML = (classes) => {
...
 										{row.Algorithm}
 									</TableCell>
-									<TableCell align="right">
+									<TableCell align="center">
 										{row.Word}
 									</TableCell>
-									<TableCell align="right">
-										{(row.Percentage + 100).toFixed(0)} %
+									<TableCell align="center">
+										{(row.Percentage * 100).toFixed(0)} %
```
Comment (jchoque87) [inline ReactUI/src/ui/components/materialUI/machine-learning/TableML.jsx:42]: if the Algorithm return 0.999998 what's is the answer

**Review by jchoque87 — APPROVED** [review]

**Pull request merged**